test/razorpay/test_payment.rb

```diff
@@ -23,6 +26,111 @@ def test_payments_should_be_available
     end
 
+    def test_payments_fetch_downtime
+      stub_get( %r{payments/downtimes$}, 'downtimes_collection')
+      payments = Razorpay::Payment.fetch_payment_downtime
+      assert_instance_of Razorpay::Collection, payments, 'Payments should be an array'
+      refute_empty payments.items, 'payments should be more than one'
+    end
+    
+    def test_payments_fetch_downtime_by_id
+      stub_get( %r{payments/downtimes/#{@downtime_id}$}, 'fake_downtime')
+      payment = Razorpay::Payment.fetch_payment_downtime_by_id(@downtime_id)
+      assert_instance_of Razorpay::Entity, payment, 'Payment not an instance of Razorpay::Payment class'
+      assert_equal @downtime_id , payment.id
+    end
+    
+    def test_payments_fetch_card_details
+      stub_get( %r{payments/#{@payment_id}/card$}, 'fake_card')
+      card = Razorpay::Payment.fetch_card_details(@payment_id)
+      assert_instance_of Razorpay::Card, card, 'Card not an instance of Razorpay::Card class'
+      assert_equal @card_id , card.id
+    end 
+
+    def test_payments_fetch_multiple_refund_for_payment
+      stub_get( %r{payments/#{@payment_id}/refunds$}, 'refund_collection',{})
+      payments = Razorpay::Payment.fetch_multiple_refund(@payment_id,{})
+      assert_instance_of Razorpay::Collection, payments, 'Payment not an instance of Razorpay::Payment class'
+      assert !payments.items.empty?, 'payments should be more than one'
+    end
+
+    def test_payments_fetch_transfers
+      stub_get(%r{payments/#{@payment_id}/transfers$}, 'transfers_collection')
+      transfers = Razorpay::Payment.fetch(@payment_id).fetch_transfer
+      assert_instance_of Razorpay::Collection, transfers, 'Transfers should be an array'
+      assert !transfers.items.empty?, 'transfers should be more than one'
+    end 
+
+    def test_payments_post_transfers
+      param_attr = {
+        "transfers": [
+          {
+            "account": "acc_CPRsN1LkFccllA",
+            "amount": 100,
+            "currency": "INR",
+            "notes": {
+              "name": "Gaurav Kumar",
+              "roll_no": "IEC2011025"
+            },
+            "linked_account_notes": [
+              "roll_no"
+            ],
+            "on_hold": true,
+            "on_hold_until": 1671222870
+          },
+          {
+            "account": "acc_CNo3jSI8OkFJJJ",
+            "amount": 100,
+            "currency": "INR",
+            "notes": {
+              "name": "Saurav Kumar",
+              "roll_no": "IEC2011026"
+            },
+            "linked_account_notes": [
+              "roll_no"
+            ],
+            "on_hold": false
+          }
+        ]
+      }
+
+      stub_post(%r{payments/#{@payment_id}/transfers$}, 'fake_transfer', param_attr.to_json)
+      payment = Razorpay::Payment.fetch(@payment_id).transfer(param_attr.to_json)
+      assert_instance_of Razorpay::Payment, payment, 'Payment not an instance of Razorpay::Payment class'
+      assert_equal 'captured', payment.status
```

Comment:
`test_payments_post_transfers` testcase failed. Instance is related to transfer not payment. check and update accordingly